packages/logging/src/internal/pino.ts

```diff
@@ -125,14 +153,11 @@ const toStream = (
 )
 
 const formatJsonLog = (object: object): object => {
```

Comment:
type JsonLogObject = Record<string, unknown> & { excessArgs: unknown[] }
```suggestion
const formatJsonLog = (object: JsonLogObject): Record<string, unknown> => {
```

Reply:
Fixed